app/templates/courses/teacher-courses-view.jade

```diff
@@ -114,3 +114,14 @@ mixin course-info(course)
         a.guide-btn.btn.btn-primary(disabled=disableButtons href=("/teachers/course-solution/" + course.id + "/python") data-course-id=course.id data-course-name=course.get('name') data-event-action="Classes Guides Guide Python" class=(me.isTeacher() || me.isAdmin() ? '': 'disabled'))
           span(data-i18n="courses.view_guide_online")
           |  &mdash; Python
+
+    if view.courseChangeLog[course.id] && Object.keys(view.courseChangeLog[course.id]).length > 0
```

Comment:
Can combine as `Object.keys(view.courseChangeLog[course.id] || []).length > 0` to be less verbose.  Will be nice to have [optional chaining](https://github.com/tc39/proposal-optional-chaining) someday :)